pytential/symbolic/mappers.py

```diff
@@ -290,6 +290,10 @@ def _default_dofdesc(self, dofdesc):
                 dofdesc = dofdesc.copy(geometry=self.default_target)
             else:
                 dofdesc = dofdesc.copy(geometry=self.default_source)
+        elif dofdesc.geometry is prim.DEFAULT_SOURCE:
+            dofdesc = dofdesc.copy(geometry=self.default_source)
+        elif dofdesc.geometry is prim.DEFAULT_TARGET:
+            dofdesc = dofdesc.copy(geometry=self.default_target)
```

Comment:
@inducer, this is my naive attempt at fixing the issue with tagging, but it seems to fail the test suite.
Any ideas on a better fix?

Reply:
`DEFAULT_SOURCE` and `DEFAULT_TARGET` are just some tags (with prettier printing?), so they don't meant "auto-tag me" in most of the code. Maybe that's what doesn't work?

I would expect that the code in `system_utils` could just use `int_g.target.to_stage1()` or something like that and then autotagger should fill in the rest. Is that not enough?

Reply:
Fixed in 411e3891eae5d39b396518ca16008e7b554b2621 with a new object called `TAG_WITH_DEFAULT_SOURCE` which tells the tagger to tag with the default source.

Reply:
Sounds good!